lua/lint/linters/solhint.lua

```diff
@@ -0,0 +1,61 @@
+local vim = vim
```

Comment:
```suggestion
```

This shouldn't be needed

Reply:
guess that was from a time where I had no vim lsp support will be removed.